### PR TITLE
CoIoT is using UDP as the protocol

### PR DESCRIPTION
**Pull request by BerendWouters**
## Proposed change
The documentation doesn't describe that the port should be exposed via udp protocol instead of tcp.

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/shelly.markdown
Comment (frenck) [inline source/_integrations/shelly.markdown]: This is not a normal use case, but an advanced networking case. An average user reading will be confused or has no idea what to do with this, they won't be able to apply it on their Home Assistant OS-based system (which is the bigger majority of our users).

IMHO, we should not add this for that reason.

User managing and creating firewall, should know how they handle their traffic. The ports are extensively documented in the documentation already, so that is not an open end that is being solved.

Comment (thecode) [reply source/_integrations/shelly.markdown]: I was waiting for another review because I had the same feeling it is to confusing for most users, I think if we change:
```
In some cases, it may be needed to customize the CoAP port
```
to:
```
In some cases, it may be needed to customize the CoAP UDP port
```
It may be enough not to confuse average users, but still allow advanced users to understand they need to map UDP port.

Comment (frenck) [reply source/_integrations/shelly.markdown]: I think that makes sense @thecode

Comment (BerendWouters) [reply source/_integrations/shelly.markdown]: Good point regarding the advanced level of networking. In the end, the issue was also quickly resolved by just googling the protocol of CoAP.
The suggestion made by @thecode would've probably saved me 5 minutes and a quick search.

Comment (frenck) [reply source/_integrations/shelly.markdown]: Agree, could you adjust the PR accordingly, @BerendWouters?

Comment (BerendWouters) [reply source/_integrations/shelly.markdown]: I've changed the PR.

**Review by thecode — APPROVED** [review]
Thanks!

**Pull request merged**
